integration/kubernetes/helm-chart/monitor/source/prometheus/prometheus.yml

```diff
@@ -0,0 +1,131 @@
+#
```

Comment:
a general comment is let's use `.yaml` for all file extensions instead of `.yml`. I have no preference but this is just to keep consistent.

Reply:
The default prometheus image use .yml, I'm not sure weather the .yaml is ok, I'will test this.